www.plain.com/docs/_next/static/chunks/pages/_sites/[subdomain]/[[...slug]]-79e8424968849281.js

```
@@ -0,0 +1 @@
+!function(){try{var e="undefined"!=typeof window?window:"undefined"!=typeof global?global:"undefined"!=typeof self?self:{},n=Error().stack;n&&(e._sentryDebugIds=e._sentryDebugIds||{},e._sentryDebugIds[n]="1e03afbc-63cf-4cc2-9e65-24e84117df27",e._sentryDebugIdIdentifier="sentry-dbid-1e03afbc-63cf-4cc2-9e65-24e84117df27")}catch(e){}}(),(self.webpackChunk_N_E=self.webpackChunk_N_E||[]).push([[4742],{51999:function(e,n,t){(window.__NEXT_P=window.__NEXT_P||[]).push(["/_sites/[subdomain]/[[...slug]]",function(){return t(35949)}])},35949:function(e,n,t){"use strict";t.r(n),t.d(n,{__N_SSG:function(){return u}});var f=t(99346),u=!0;n.default=f.Z}},function(e){e.O(0,[9774,2182,8934,6202,9097,7669,7610,40,2888,179],function(){return e(e.s=51999)}),_N_E=e.O()}]);
```

Biome (1.9.4)

[error] Redundant use strict directive.

The entire contents of JavaScript modules are automatically in strict mode, with no statement needed to initiate it.
Safe fix: Remove the redundant use strict directive.


(lint/suspicious/noRedundantUseStrict)